FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -184,12 +188,12 @@ extension ComposeViewController {
             to: [email],
             cc: [],
             bcc: [],
-            from: dataManager.currentUser()?.email ?? "",
+            from: dataManager.email ?? "",
```

Comment:
Actually here it would be better to get an error that shows to the user. Is there a way to do that easily? At this stage, email must be set, and if it's not, user should know about it. Such errors should not be ignored.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/140